addons/website_sale/controllers/delivery.py

```diff
@@ -160,6 +160,8 @@ def express_checkout_process_delivery_address(self, partial_delivery_address):
         :rtype: dict
         """
         order_sudo = request.website.sale_get_order()
+        if not order_sudo:
+            return {}
```

Comment:
I know that in the docstring the return type is dict but if you look at the real return type it would be list, could you please change it to list?
```suggestion
            return []
```